README.rst

```diff
@@ -0,0 +1,16 @@
+BAP IDA Python
+==============
+
+This package provides the necessary IDAPython scripts required for
+interoperatibility between BAP and IDA Pro.
+
+Installation
+------------
+
+All of the requisite scripts can be installed via ``pip`` using::
+
+    pip install git+git://github.com/BinaryAnalysisPlatform/bap-ida-python
+
+Now all that remains is to inform IDA about where the scripts are and how to access BAP. This process is simplified greatly using the plugin installer. The installer can be run using::
+
+    python -c 'from bap_ida_python.loader.installer import main; main()'
```

Comment:
can we actually pack this into a python script?
Also, is it possible to do this in a process of `pip install` or it is an interactive script?